### ISSUE TITLE
How many iterations of the model can you get the best performance in the benchmark？

### ISSUE THREAD

**Issue by DyCheer** [bytedance/RLFN#1]
Benchmark like Set5, Set14 and  so on.
According to the paper, the initial learning rate is 5e-4, and is halved every 2e5 iterations, but the total number of iterations is not specified.

**Comment by fyan111** [comment]
We train 1000 epochs at each stage and halve learning rate every 200 epoch.
